README.md

```diff
@@ -379,6 +379,7 @@ Global Flags:
 * `make test` to run tests
 * `make lint` to lint the source code (included generated code)
 * `make release` to run one last check before opening a PR
+* `make compile version=RELEASE_TAG` to generate binaries
```

Comment:
Would you mind adding a small section to the README about using binaries and that a script is in progress?

Reply:
You may want to add that it is a known issue to click download on the release:
https://github.com/hashicorp/terraform/issues/23033

Reply:
Turns out curl bypasses this:
```
curl -L https://github.com/coinbase/rosetta-cli/releases/download/v0.4.1/rosetta-cli-0.4.1-darwin-10.6-amd64 -o rosetta-cli; chmod +x rosetta-cli;
```

Reply:
You may want to include this in the README ^^ with a warning to not download on the UI